src/tools/schema2kotlin.ts

```diff
@@ -107,7 +107,7 @@ class KotlinGenerator implements ClassGenerator {
   fieldsReset: string[] = [];
   getUnsetFields: string[] = [];
 
-  constructor(readonly node: SchemaNode, private opts: minimist.ParsedArgs) {}
+  constructor(readonly node: SchemaNode, private readonly opts: minimist.ParsedArgs) {}
```

Comment:
What's the linter's justification for this? Is it just that we don't write to it so we should be restrictive by default?

Thanks @csilvestrini

Reply:
That's right